packages/tezos-ts-rpc/src/types.ts

```diff
@@ -16,6 +16,33 @@ export interface ContractResponse {
   counter: string;
 }
 
+export interface ConstantsResponse {
+  proof_of_work_nonce_size: number;
+  nonce_length: number;
+  max_revelations_per_block: number;
+  max_operation_data_length: number;
+  preserved_cycles: number;
+  blocks_per_cycle: number;
+  blocks_per_commitment: number;
+  blocks_per_roll_snapshot: number;
+  blocks_per_voting_period: number;
+  time_between_blocks: BigNumber[];
```

Comment:
It's good that we put the `BigNumber` type there however since Typescript do not do implicit type conversion it does not guaranty that the RPC response will be of that type, in fact it is probably a `string`.

That means that we need to convert the json returned by the RPC to match this type see: https://github.com/ecadlabs/tezos-ts/blob/master/packages/tezos-ts-rpc/src/tezos-ts-rpc.ts#L111

In that example the property balance is a BigNumber so we convert the balance property from a json string to a BigNumber doing:
```
return {
   ...contractResponse,
   balance: new BigNumber(contractResponse.balance)
};
```

I suggest that we make a unit test with a real json response and verify that our getConstant actually convert string to big number like it is done here: https://github.com/ecadlabs/tezos-ts/blob/master/packages/tezos-ts-rpc/test/tezos-ts-rpc.spec.ts#L33

Example getConstants response that we can use for the test
```json
{"proof_of_work_nonce_size":8,"nonce_length":32,"max_revelations_per_block":32,"max_operation_data_length":16384,"max_proposals_per_delegate":20,"preserved_cycles":3,"blocks_per_cycle":2048,"blocks_per_commitment":32,"blocks_per_roll_snapshot":256,"blocks_per_voting_period":8192,"time_between_blocks":["30","40"],"endorsers_per_block":32,"hard_gas_limit_per_operation":"400000","hard_gas_limit_per_block":"4000000","proof_of_work_threshold":"70368744177663","tokens_per_roll":"10000000000","michelson_maximum_type_size":1000,"seed_nonce_revelation_tip":"125000","origination_size":257,"block_security_deposit":"128000000","endorsement_security_deposit":"16000000","block_reward":"0","endorsement_reward":"0","cost_per_byte":"1000","hard_storage_limit_per_operation":"60000"}
```